### ISSUE TITLE
Karma fails to load rxjs modules

### ISSUE THREAD

**Issue by LuxDie** [angular/angular-cli#131]
After running `ng new`, `ng build` and `node_modules/karma/bin/karma start`, Karma is missing 4 modules.

Adding these lines to `karma-conf.js` fixes it (the first one is a replacement of `testing.js`, which doesn't exist):
```javascript
{pattern: 'node_modules/angular2/bundles/testing.dev.js', included: true, watched: true},
{pattern: 'node_modules/rxjs/bundles/rx.js', included: true, watched: true},
{pattern: 'node_modules/reflect-metadata/reflect.js', included: true, watched: true},
{pattern: 'node_modules/zone.js/dist/zone.js', included: true, watched: true},
```

I'm guessing this happens because I'm using npm 3 so my node_modules are flattened.

**Comment by cironunes** [comment]
Thanks for reporting. This is a known issue and should be addressed by #112

**Comment by cironunes** [comment]
Fixed.
